routing/src/rib/vrf.rs

```diff
@@ -279,6 +280,42 @@ impl Vrf {
         }
     }
 
+    /////////////////////////////////////////////////////////////////////////
+    /// Check if a [`Vrf`] needs to be deleted and mark it as such. Only
+    /// [`Vrfs`] in state `Deleting` can be deleted and the default VRF never
+    /// gets to that status.
+    /////////////////////////////////////////////////////////////////////////
+    pub fn check_deletion(&mut self) {
+        if self.status == VrfStatus::Deleting {
+            let one = NonZeroUsize::new(1).unwrap_or_else(|| unreachable!());
+            if self.routesv4.len() == one && self.routesv6.len() == one {
```

Comment:
Why do we return NonZeroUsize from len() instead of usize?  Can there never be 0 routes in the data structure?